### PR TITLE
Ingest url data source

### PR DESCRIPTION
**Pull request by AlexisJasso**
https://firehydrant.atlassian.net/browse/PDE-1965

This adds the ingest URL as a data source.  See included docs for options and usage.

### CODE REVIEW

**Comment by davidcelis** [comment]
@AlexisJasso alongside the `/v1/signals/ingest_key` endpoint, there's a `/v1/signals/transposers` endpoint that takes the same params and lists transposer URLs based on what's provided (team ID, policy ID, schedule ID, or user ID). you could use that to get around the URL hackery?

**Comment by AlexisJasso** [comment]
> @AlexisJasso alongside the `/v1/signals/ingest_key` endpoint, there's a `/v1/signals/transposers` endpoint that takes the same params and lists transposer URLs based on what's provided (team ID, policy ID, schedule ID, or user ID). you could use that to get around the URL hackery?

Wait... why isn't that documented?  There are no params listed!

**Review by wilsonehusin — APPROVED** [review]
functionality looks good — one thing about docs that we can probably clarify, but otherwise OK!

File: docs/data-sources/ingest_url.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  encode_as      = "json"
+  # ...
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+> You must provide one of these options.
```
[the remark this answers is not in the document]
Comment (AlexisJasso) [reply docs/data-sources/ingest_url.md]: LOL.  Yep.  All attributes are optional.  Docs fixed.

**Pull request merged**